backend/docker-dev.sh

```diff
@@ -2,6 +2,9 @@
 if [ $1 == "up" ]
 then
   docker-compose -f docker-compose.dev.yml up -d
+  # need to add volume so we can copy the clowder theme into the right place inside the container
```

Comment:
why not just add the volume to the docker compose? with this approach if anyone runs the default docker compose to run the full stack it will not include this.